server/history/util.py

```diff
@@ -20,8 +21,10 @@
 
 
 def get_history_db_connection(with_db_name=True):
+    db_host = os.getenv("POSTGRES_HOST", "localhost")
```

Comment:
I did this as I was told the default value for Postgres is localhost and that this file is used for deployments.

Reply:
Wouldn't that be postgres, from [docker-compose.yml L12](https://github.com/transitmatters/new-train-tracker/pull/90/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R12)

Reply:
this is a setup that if the POSTGRES_HOST isn't present it defaults to localhost.